lib/default.nix

```diff
@@ -1,8 +1,9 @@
-/* Library of low-level helper functions for nix expressions.
- *
- * Please implement (mostly) exhaustive unit tests
- * for new functions in `./tests.nix`.
- */
+/**
+  Library of low-level helper functions for nix expressions.
+
+  Please implement (mostly) exhaustive unit tests
```

Comment:
Is this an implementation comment? If yes then we should move it into one.

Reply:
this definitely does not belong in the nix manual, it should not be a doc comment.